lib/routes.js

```diff
@@ -56,10 +60,13 @@ app.use('/', pairingRoutes)
 app.use(findOperatorId)
 app.use(populateDeviceId)
 app.use(computeSchema)
-if (!devMode) app.use(authorize)
+app.use(authorize)
 app.use(configRequiredRoutes, populateSettings)
 app.use(filterOldRequests)
 
+app.use('/graphql', recordPing)
+graphQLServer.applyMiddleware({ app })
```

Comment:
Does the order matter here? I'd personally prefer the routes to be grouped up together, but it's a nitpick

Reply:
Hmm not sure, I'll try.

Reply:
Seems to work.